docs/src/analysis/sensitivity.md

```diff
@@ -383,3 +385,196 @@ end
+arrays for each paramter and dependent variable pair.
+
+We can plot the trajectory of the sensitivity with the standard deviation bars.
+```julia
+# For the first parameter (a)
+stdv1 = sqrt.(m.variances[1])
+p = plot(m.means[1]', yerror=stdv1)
+```
+![morrisparameter1](../assets/morris1.png)
+
+```julia
+# For the second parameter (b)
+stdv2 = sqrt.(m.variances[2])
+p = plot(m.means[2]', yerror=stdv2)
+```
+![morrisparameter2](../assets/morris2.png)
+
+For Sobol Method
+
+```julia
+
+s0 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,0)
+Out[8]: 2-element Array{Array{Float64,2},1}:
+ [NaN 0.507831 … 1.00731 1.00436; NaN 1.92336 … 0.732384 0.730945]  
+ [NaN 0.47214 … 0.676224 0.681525; NaN -1.68656 … 0.879557 0.877603]
+
+s1 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,1)
+Out[9]: 2-element Array{Array{Float64,2},1}:
+ [NaN 0.39537 … 0.341697 0.343645; NaN -2.06101 … 0.10922 0.106976]     
+ [NaN 0.652815 … 0.00910675 0.00815206; NaN 5.24832 … 0.296978 0.296639]
+
+s2 = sobol_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],N,2)
+Out[10]: 1-element Array{Array{Float64,2},1}:
+ [NaN -0.0596478 … 0.652303 0.657847; NaN -1.84504 … 0.645139 0.620036]
+```
+We can decide which order of Sobol Indices we are interested in my passing an argument for it, 
+by default it gives the second order indices. Again the result is obtained over the entire `timespan`
+
+We plot the first order and total order Sobol Indices for some timepoints for each of the parameters (`a` and `b`).
+
+``julia
+p1 = bar(["a","b"],[s0[1][end-2],s0[2][end-2]],color=[:red,:blue],title="Total Order Indices for 199th time point",legend=false)
+p2 = bar(["a","b"],[s1[1][end-2],s1[2][end-2]],color=[:red,:blue],title="First Order Indices for 199th time point",legend=false)
+p3 = bar(["a","b"],[s0[1][3],s0[2][3]],color=[:red,:blue],title="Total Order Indices for 2nd time point",legend=false)
+p4 = bar(["a","b"],[s1[1][3],s1[2][3]],color=[:red,:blue],title="First Order Indices for 2nd time point",legend=false)
+plo = plot(p1,p2,p3,p4,layout=(4,1),size=(600,500))
+```
+![sobolplot](../assets/sobolbars.png)
+
+Here we plot the Sobol indices of first order and the total Sobol indices for the parameters `a` and `b`. The plots are obtained by getting the Sobol Indices at the 199th and the 2nd time point of the first dependent variable `x` from the 200-length sensitivities over the entire time span. The length of the bar represents the quantification of the sensitivity of the output to that parameter and hence for the 199th time point you can say that `x` is more sensitive to `b`, also you can observe how the relative difference between `a` and `b` is larger in the first order than the total order indices, this tells us that most of the contribution of `a` to `x` arises from interactions and it's individual non-interaction contribution is significantly lesser than `b` and vice-versa for `b` as it's first order plot indicates quite high value.
```

Comment:
>sensitivity of the output

of what output? The output has two values.

Reply:
>The length of the bar represents the quantification of the sensitivity of the output to that parameter and hence for the 199th time point you can say that `x` is more sensitive to `b`,

I have mentioned `x`?

Reply:
aha. I didn't see that. I would say `x(t)` at `t = ...`. Then it looks good and we can merge.